packages/cashier/src/pages/withdrawal/__tests__/withdrawal.spec.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
```

Comment:
I guess there are no types used here, we don’t need .tsx for this file

Reply:
I'm testing the `withdrawal.tsx` file and the `.spec.tsx` has react components in it, that is why we use withdrawal.spec.tsx here instead of `.ts` or `.js` or `.jsx`